packages/sdk/src/evm/common/feature-detection/resolveContractUriFromAddress.ts

```diff
@@ -52,6 +52,12 @@ export async function resolveImplementation(
   }
 
   // check other proxy types
+  const beacon = await getBeaconFromStorageSlot(address, provider); // if the contract is BeaconProxy
```

Comment:
would be nice to parallelize this with `fetchBytecode` - you can promise all both

Reply:
Updated